shared/docs/flows/AppHash.md

```diff
@@ -1,40 +1,122 @@
-# AppHash
+# AppHash <!-- omit in toc -->
+
+- [Context Initialization](#context-initialization)
+- [Block Application](#block-application)
+- [Block Commit](#block-commit)
+
+<!-- See if there's an answer in this question to add links to notes: https://stackoverflow.com/questions/74103729/adding-hyperlinks-to-notes-in-mermaid-sequence-diagrams -->
```

Comment:
👍

Reply:
@deblasis Do you think this needs any other changes?